### PR TITLE
Indicate process of app initialization

### PR DESCRIPTION
**Pull request by hsw**
Issue #325 and #324

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,9 +8,46 @@
...
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-md-9">
+          <div class="content">
+            <div class="box">
+              <noscript>
+                <div class="alert alert-danger" role="alert">
+                  <strong>Javascript отключен!</strong> Этот сайт, пока ещё, требует JS для работы, извините.
```
Comment (n1313) [inline public/index.html]: "пока еще" не должно отделяться запятыми

Comment (epicmonkey) [inline public/index.html]: English please.

File: public/index.html [hunk 2]
```diff
@@ -8,9 +8,46 @@
...
   <link rel="icon" href="/img/favicon.png" type="image/x-icon" />
 </head>
 <body>
+  <div id="ember-app">
+    <div class="container">
+      <div class="row header-row">
+        <div class="col-md-4">
+          <div class="header">
+            <h1 class="title">
+              <a href="/">freefeed</a>
```
Comment (clbn) [inline public/index.html]: This should be "Pepyatka", I suppose. "FreeFeed" labeling should go to `FreeFeed/pepyatka-html` repo.

File: public/index.html (hunk 1 above)
Comment (clbn) [inline public/index.html]: Let's rewrite this copy in a more formal way (also, in English).

File: public/index.html (hunk 2 above)
Comment (epicmonkey) [inline public/index.html]: Just a heads up: we must secure^W not use direct links (e.g. `<a href="/">`) unless there are stong reasons to do this. Instead there are ember helpers like `{{link-to}}` or `<a {{action whatever}}>Link</a>` that work smoothly with SPA.

Comment (clbn) [inline public/index.html]: ^ Also, THIS!

Comment (hsw) [inline public/index.html]: It's static html file. Helpers is not supported.

File: public/index.html [hunk 3]
```diff
@@ -8,9 +8,44 @@
...
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-md-9">
+          <div class="content">
+            <div class="box">
+              <noscript>
+                <div class="alert alert-danger" role="alert" id="js-disabled-warn"><strong>Javascript disabled!</strong> Looks like you have JavaScript disabled in your browser. Please enable it to enjoy our service.</div>
```
Comment (n1313) [inline public/index.html]: Javascript *is* disabled.

File: public/index.html (hunk 2 above)
Comment (epicmonkey) [inline public/index.html]: Gosh! You are right. I was thinking of rendering a static file like Facebook does with transparent/empty posts, though it needs to be thought through first. First version is better than nothing.

**Comment by hsw** [comment]
labeling: ansible replace `"Pepyatka"` to `"{{ pepyatka_service_name }}"` in index.html:
https://github.com/pepyatka/pepyatka-ansible/blob/master/roles/nginx/tasks/main.yml#L96

**Pull request merged**